edge-hub/src/Microsoft.Azure.Devices.Edge.Hub.Service/Hosting.cs

```diff
@@ -30,7 +31,11 @@ public class Hosting
                 {
                     options.Listen(!Socket.OSSupportsIPv6 ? IPAddress.Any : IPAddress.IPv6Any, port, listenOptions =>
                     {
-                        listenOptions.UseHttps(serverCertificate);
+                        listenOptions.UseHttps(serverCertificate, (connectionAdapterOptions) =>
+                        {
+                            connectionAdapterOptions.ClientCertificateMode = ClientCertificateMode.AllowCertificate;
+                            connectionAdapterOptions.ClientCertificateValidation = (clientCert, chain, policyErrors) => true;
+                        });
```

Comment:
So, if we do this, will the cert actually be "verified" (whether it is correct, matches, the chain, etc)?

Reply:
Verification will happen in authenticate async in all cases. Essentially we have taken validation out of the TLS (or transport layer) and brought it up to the application layer.

Reply:
I can add a comment to mention this.